tests/config.neon

```diff
@@ -20,6 +21,13 @@ parameters:
             error: BrandEmbassyTest\Slim\Dummy\ApiErrorHandler
 
         routes:
+            "app":
+                "":
+                    "":
+                        get:
+                            service: BrandEmbassyTest\Slim\Dummy\GetHelloWorldAction
```

Comment:
Test na toto je pak že se  `/app` routuje správně sem. Původně by to bylo `/app/`